src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -676,6 +677,11 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             'Populate the target framework combobox
             PopulateTargetFrameworkComboBox(Me.TargetFramework)
+
+            ' Hide the AssemblyInformation button if project supports "Pack" capability, and hence has a Package property page with assembly info properties.
+            If Me.ProjectHierarchy.IsCapabilityMatch("Pack") Then
```

Comment:
There is a constant in our Core Managed project. However, it isn't referenced by this project. I'll define another constant for this project.